.flowconfig

```diff
@@ -9,10 +9,12 @@
 [include]
 
 [libs]
+flow-libs
 
 [options]
 suppress_comment=.*\\$FlowFixMe.*
 suppress_comment=.*\\$FlowIssue.*
+suppress_comment=\\(.\\|\n\\)*\\$FlowDisableNextLine
 
 [version]
-^0.68.0
+^0.75.0
```

Comment:
The most recent flow version is 0.89.0. Is there a reason you don't want to use the most recent version?

Reply:
Reason for updating to 0.75 is to avoid breaking changes to users relying on bundled flow 0.68. I was planning to use v0.89 in a separate commit. Anyways, we released v1.0 so it's fine.